internal/gitstream/assign.go

```diff
@@ -103,12 +108,17 @@ func (a *Assign) assignIssues(ctx context.Context) error {
 
 			logger.Info("Assigning issue")
 
+			var userLogin string
 			user, err := a.UserHelper.GetUser(ctx, upstreamCommit)
 			if err != nil {
-				return fmt.Errorf("could not get the upstream commit author for downstream issue %d: %v", *issue.Number, err)
+				logger.Info("WARNING: could not get the upstream commit author for downstream issue, picking a random assignee",
+					"issue", *issue.Number)
+				userLogin = "notanowner"
```

Comment:
This feels a bit hacky. I would instead add a `getRandomAssignee` method. WDYT?

Reply:
You mean that that method will return a random string?
I don't mind but what is the difference? Feels like the chances we have an approver m/s called `notanowner` is less likely that having a random name in the approvers don't you think so?